sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -31148,6 +31148,154 @@
                   "fieldSet": false,
                   "id": "efqfe2q"
                 },
+                {
+                  "autofocus": false,
+                  "input": true,
+                  "tableView": false,
+                  "inputType": "text",
+                  "inputMask": "",
+                  "label": "Please estimate your weekly transportation costs.",
+                  "key": "studentDataAdditionalTransportCost",
+                  "placeholder": "",
+                  "prefix": "",
+                  "suffix": "",
+                  "defaultValue": "",
+                  "protected": false,
+                  "persistent": true,
+                  "hidden": false,
+                  "clearOnHide": true,
+                  "delimiter": true,
+                  "decimalLimit": 2,
+                  "requireDecimal": true,
+                  "validate": {
+                    "required": true,
+                    "multiple": "",
+                    "custom": ""
+                  },
+                  "conditional": {
+                    "show": "true",
+                    "when": "strongYouMayBeEligibleForAnAdditionalTransportationAllowanceIfThereIsNoLimitedLocalPublicTransitWhereYouLiveOrIfYouAreRequiredDueToSpecialCircumstancesDoYouWantToBeAssessedForAdditionalTransportationCostsStrong",
+                    "eq": "yes"
+                  },
+                  "type": "currency",
+                  "labelPosition": "top",
+                  "tags": [],
+                  "properties": {},
+                  "lockKey": true
+                },
+                {
+                  "autofocus": false,
+                  "input": true,
+                  "tableView": false,
+                  "inputType": "number",
+                  "label": "Please estimate your weekly travel distance in kilometers:",
+                  "key": "studentDataAdditionalTransportKm",
```

Comment:
The `studentData` prefix is added while loading the values on Camunda.
For form.io API keys, we can use the variable names provided in the ticket without the prefix.
- additionalTransportKm
- additionalTransportCost
- additionalTransportWeeks
- additionalTransportPlacement